spec/lib/apress/utils/email_validation_spec.rb

```diff
@@ -19,10 +19,10 @@
   end
 
   context 'when email is valid' do
-    it_behaves_like('valid email', %w{valid.email@email.com va-l-id_email@email.com validemail@email.com valid.email@e-mail.com valid_email@e.mail.com v3alid_email01@e.mail.com })
+    it_behaves_like('valid email', %w{моя@почта.рф valid.email@email.com va-l-id_email@email.com validemail@email.com valid.email@e-mail.com valid_email@e.mail.com v3alid_email01@e.mail.com })
```

Comment:
Metrics/LineLength: Line is too long. [192/120]<br>Layout/SpaceInsidePercentLiteralDelimiters: Do not use spaces inside percent literal delimiters.